lib/ProductOpener/Display.pm

```diff
@@ -7423,7 +7423,7 @@ JS
 				$icon = "<span style=\"font-size:32px;margin-right:0.2em;vertical-align:middle;line-height:24px;\">". display_icon($icon) ."</span>";
 			}
 
-			$html_analysis .= "<span class=\"alert round label\" style=\"background-color:$color;color:white;font-size:1rem;padding-right:1em;\">"
+			$html_analysis .= "<span class=\"alert round label ingredients_analysis\" style=\"background-color:$color;color:white;font-size:1rem;padding-left:0.2rem;padding-right:1rem;\">"
```

Comment:
Can we put the `padding left`, `padding right`, `color` and `font-size` in the scss?

Reply:
Of course. Good catch!